src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -78,19 +77,19 @@
      * Public constructor.
      *
      * @param actorSystem Instance of <code>ActorSystem</code>.
-     * @param metricsFactory Instance of <code>MetricsFactory</code>.
+     * @param metrics Instance of <code>PeriodicMetrics</code>.
      * @param healthCheckPath The path for the health check.
      * @param statusPath The path for the status.
      * @param supplementalRoutes List of supplemental routes in priority order.
      */
     public Routes(
             final ActorSystem actorSystem,
-            final MetricsFactory metricsFactory,
+            final PeriodicMetrics metrics,
```

Comment:
^ Based on discussion should take PM because PM will be created from MF; note, that PM interface will be renamed to M (eventually, possibly only in ISM).